### ISSUE TITLE
[warning] initialize_all_variables is deprecated.

### ISSUE THREAD

**Issue by Samurais** [Conchylicultor/DeepQA#25]
# Description
Runing the current code against tensorflow 0.12.0-rc0.
```
initialize_all_variables (from tensorflow.python.ops.variables) is deprecated.
```

# Solution
```
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 0:   Y                                                                                        [26/579]
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GRID K520, pci bus id: 0000:00:03.0)

WARNING:tensorflow:From /home/ubuntu/git/chatbot_butterfly/chatbot/chatbot.py:173 in main.: initialize_all_variables (from tensorflow.python.ops.
variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.global_variables_initializer` instead.
Training:   0%|          | 3/4030 [00:00<22:14,  3.02it/s]I tensorflow/core/common_runtime/gpu/pool_allocator.cc:247] PoolAllocator: After 1768 get requests,
put_count=1460 evicted_count=1000 eviction_rate=0.684932 and unsatisfied allocation rate=0.79638
```

**Comment by zameran** [comment]
Hello! See pull request here https://github.com/Conchylicultor/DeepQA/pull/26
Hope it helps.

**Comment by Samurais** [comment]
Moment ago, I changed back to 0.11. As it turns out, **model.ckpt** is not generated with 0.12.
Check out #23. So, if we really want to use 0.12, we should better fix this problem first.

**Comment by zameran** [comment]
Looks like we need to migrate some stuff to tensorflow's 12 API.
But i don't have any warnings...

@Samurais

**Comment by Conchylicultor** [comment]
initialize_all_variables is just a wrapper around global_variables_initializer: https://github.com/tensorflow/tensorflow/blob/20c3d37ecc9bef0e106002b9d01914efd548e66b/tensorflow/python/ops/variables.py#L1170

There have been change with `scalar_summary` > `summary.scalar` as well. As explain in #26, this change is too recent to change the program, but I'll certainly fix that in the future.

**Comment by Conchylicultor** [comment]
It should be solved with #52

**Comment by CG1507** [comment]
I had same problem before I've solved using: 
try: 
     #code  
except:
     pass
